src/pages/index.astro

```diff
@@ -146,13 +232,145 @@ section.hero {
   flex: 1 1 20rem;
 }
 
+.audienceSection {
+  margin-inline: var(--spacing-md);
+}
+
+.audience-grid {
+  display: grid;
+  grid-template-columns: 1fr 0.25rem 1fr;
+  gap: var(--spacing-xs);
+  align-items: start;
+  margin-bottom: var(--spacing-xs);
+}
+
+.audience-grid > div {
+  width: 100%;
+  height: 100%;
+}
+
+.audience-card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var(--color-white);
+  border-radius: var(--pill-rounding);
+  max-width: 100%;
+  justify-content: space-between;
+}
+
+.audience-card.developer {
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+}
+
+.audience-card .btn, .audience-card .media-cta {
+  margin-top: var(--spacing-3xl);
+}
+
+.audience-card.developer div {
+  flex-basis: 50%;
+}
+
+.audience-card > div {
+  width: 100%;
+}
+
+.audience-card .btn {
+  width: fit-content;
+}
+
+.audience-img {
+  max-width: 30vw;
+  height: auto;
+  object-fit: contain;
+  display: block;
+}
+
+.active.audience-img {
+  display: none;
+}
+
+.audience-card {
+  padding: 2rem;
+  color: var(--inactive-font-color);
+}
+
+.audience-card:hover, .audience-card:focus {
+  color: var(--default-font-color);
+}
+
+.audience-card:hover .active.audience-img, .audience-card:focus .active.audience-img {
+  display: block;
+}
+
+.audience-card:hover .inactive.audience-img, .audience-card:focus .inactive.audience-img {
+  display: none;
+}
+
+.audience-card h3 {
+  margin-block-end: var(--spacing-lg);
+}
+
+.divider--vertical {
+  position: relative;
+  width: 0.25rem;
+  align-self: stretch;
+  display: flex;
+  justify-content: center;
+}
+
+.divider--vertical::before {
+  content: "";
+  position: absolute;
+  top: var(--padding-2xl);
+  bottom: var(--padding-2xl);
+  left: 0;
+  right: 0;
+  background-image: linear-gradient(
+    to top,
+    #7F76B2 12%,
+    #F69656 30%,
+    #F2797F 59%,
+    #9CD6CB 90%
+  );
+  border-radius: var(--fully-rounded);
+}
+
+.divider--horizontal {
+  position: relative;
+  height: 0.25rem;
+  width: 100%;
+  align-self: stretch;
+  display: flex;
+  align-items: center;
+  margin-bottom: var(--spacing-xs);
+}
+
+.divider--horizontal::before {
+  content: "";
+  position: absolute;
+  left: calc(12.5% + var(--padding-2xl));
+  right: calc(12.5% + var(--padding-2xl));
+  top: 0;
+  bottom: 0;
+  background-image: linear-gradient(
+    to left,
+    #7F76B2 12%,
+    #F69656 30%,
+    #F2797F 59%,
+    #9CD6CB 90%
+  );
+  border-radius: var(--fully-rounded);
+}
```

Comment:
Some of these can be merged using a `.divider` class and more CSS vars (e.g. `--gradient-direction`).